config/rtdb2_configuration.xml

```diff
@@ -10,7 +10,7 @@
      * interface       (optional)  interface overrule, 'auto' (default) will resolve to available wireless adapter
      * frequency       (mandatory) transmitter frequency (Hz)
      * port            (mandatory) port to use
-     * compression     (mandatory) boolean flag to enable compression
+     * compression     (optional)  deprecated
```

Comment:
Hm, so it would appear I applied compression at database level instead of comm.
Maybe we should devise a stress-tester. Nice to have.